gce/setup.sh

```diff
@@ -0,0 +1,30 @@
+# Start cluster on Google cloud
+gcloud container clusters create pangeo --num-nodes=10 --machine-type=n1-standard-2 --zone=us-central1-b  --cluster-version=1.8.4-gke.1
+gcloud container clusters get-credentials pangeo --zone us-central1-b --project pangeo-181919
```

Comment:
At some point, using preemptible VMs might save us lots of money
https://cloud.google.com/kubernetes-engine/docs/concepts/preemptible-vm

Reply:
I agree.  There is some delay in launching new nodes, but generally I agree that long term this is a good strategy.  Short term I'm just leaving a few nodes on for a while.  I'm judging that the cost in expense is worth it to reduce friction when getting started.  Please let me know if you feel differently and I can adjust.

Reply:
This is good for now.